src/main/java/underdogs/devbie/answer/controller/AnswerController.java

```diff
@@ -0,0 +1,76 @@
+package underdogs.devbie.answer.controller;
+
+import java.net.URI;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.AllArgsConstructor;
+import underdogs.devbie.answer.dto.AnswerCreateRequest;
+import underdogs.devbie.answer.dto.AnswerResponse;
+import underdogs.devbie.answer.dto.AnswerResponses;
+import underdogs.devbie.answer.dto.AnswerUpdateRequest;
+import underdogs.devbie.answer.service.AnswerService;
+import underdogs.devbie.auth.controller.interceptor.annotation.NoValidate;
+import underdogs.devbie.auth.controller.resolver.LoginUser;
+import underdogs.devbie.user.domain.User;
+
+@RestController
+@RequestMapping("/api/answers")
+@AllArgsConstructor
+public class AnswerController {
+
+    private final AnswerService answerService;
+
+    @PostMapping
+    public ResponseEntity<Void> save(@LoginUser User user,
+        @RequestBody @Valid AnswerCreateRequest answerCreateRequest) {
+        Long id = answerService.save(user, answerCreateRequest);
+
+        return ResponseEntity.created(URI.create(String.format("/api/answers/%d", id)))
+            .build();
+    }
```

Comment:
```suggestion
    @PostMapping
    public ResponseEntity<Void> save(@LoginUser User user,
        @RequestBody @Valid AnswerCreateRequest request) {
        Long id = answerService.save(user, answerCreateRequest);

        return ResponseEntity.created(URI.create(String.format("/api/answers/%d", id)))
            .build();
    }
```
request로 통일하는건 어떻게 생각하세요? 팀원들 모두 있을 때 이야기해볼까요?

Reply:
`Long id = answerService.save(user, answerCreateRequest);` 
큰 문제는 아니지만, answerService.save()에서 User의 id만 사용하고 있어요! 
controller에서 ID만 넘기는건 어떨까요?

Reply:
저도 앨런의견에 동의합니다!

Reply:
두분 말씀의 의도는 잘 알겠습니다!

제가 굳이 User 객체를 삽입한 이유는 user.getId()하는 시점부터 객체화된 User의 이점을 활용할 수 없다고 판단해서 인데요.

User안에서 userId만 사용한다고 하더라도 `user.match(Answer에 존재하는 작성자 userId)` 처럼 활용하는 것이 더 낫지 않을까요 ? 

물론 Answer 클래스에 User 객체에 대한 의존성이 생기는 단점이 있긴 합니다만 트레이드 오프라고 생각합니다~!

다른 분들의 의견을 듣고 싶어요! :)